### PR TITLE
Improve the logs for the cluster_claim step

### PR DESCRIPTION
**Pull request by hongkailiu**
To avoid the confusion what is going on with the job In case like https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/17734/rehearse-17734-periodic-ci-redhat-operator-ecosystem-cvp-ocp-4.7-fast-as-heck-aws/1382319773408301056#1:build-log.txt%3A10

/cc @petr-muller

### CODE REVIEW

File: pkg/steps/cluster_claim.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func (s *clusterClaimStep) run(ctx context.Context) error {
 	if err := s.hiveClient.Create(ctx, claim); err != nil {
 		return fmt.Errorf("failed to created cluster claim %s in namespace %s: %w", claim.Name, claim.Namespace, err)
 	}
-	logrus.WithField("claim.Name", claim.Name).WithField("claim.Namespace", claim.Namespace).Info("waiting for the claimed cluster to be ready ...")
+	logrus.WithField("claim.Name", claim.Name).WithField("claim.Namespace", claim.Namespace).Info("Waiting for the claimed cluster to be ready ...")
```
Comment (stevekuznetsov) [inline pkg/steps/cluster_claim.go:119]: This is a debug level log.

File: pkg/steps/cluster_claim.go [hunk 2]
```diff
@@ -116,14 +116,17 @@ func (s *clusterClaimStep) run(ctx context.Context) error {
 	if err := s.hiveClient.Create(ctx, claim); err != nil {
 		return fmt.Errorf("failed to created cluster claim %s in namespace %s: %w", claim.Name, claim.Namespace, err)
 	}
-	logrus.WithField("claim.Name", claim.Name).WithField("claim.Namespace", claim.Namespace).Info("waiting for the claimed cluster to be ready ...")
+	logrus.Info("Waiting for the claimed cluster to be ready.")
 	claim = &hivev1.ClusterClaim{}
 	if err := wait.Poll(15*time.Second, s.clusterClaim.Timeout.Duration, func() (bool, error) {
```
Comment (stevekuznetsov) [inline pkg/steps/cluster_claim.go:121]: This should really be a watch.

Comment (hongkailiu) [reply pkg/steps/cluster_claim.go:121]: I will do with another PR since this one is about logging.

Comment (hongkailiu) [reply pkg/steps/cluster_claim.go:121]: https://github.com/openshift/ci-tools/pull/1874

**Review by stevekuznetsov — APPROVED** [review]

**Pull request merged**